packages/lesswrong/components/books/Book2020Animation.tsx

```diff
@@ -0,0 +1,184 @@
+    width: SECTION_WIDTH + 100,
+    paddingTop: PADDING,
+    paddingBottom: PADDING,
+    paddingRight: PADDING,
+    [theme.breakpoints.down('sm')]: {
+      height: (HEIGHT*.75) + (PADDING * 3.5),
+    },
+    [theme.breakpoints.down('lg')]: {
+      '& $revealedContent': {
+        opacity: 1,
+      }
+    },
+    [theme.breakpoints.down('md')]: {
+      width: "100%",
+    },
+    '& $one': {
+      left: 0,
+      [theme.breakpoints.down('lg')]: {
+        left: '60px !important',
+        top: '10px !important',
+        transform: "rotate(-15deg)"
+      }
+    },
+    '& $two': {
+      left: WIDTH + PADDING,
+      [theme.breakpoints.down('lg')]: {
+        left: '75px !important',
+        top: '25px !important',
+        transform: "rotate(-5deg)"
+      }
+    },
+    '& $three': {
+      left: (WIDTH + PADDING) * 2,
+      [theme.breakpoints.down('lg')]: {
+        left: '90px !important',
+        top: '40px !important',
+        transform: "rotate(5deg)"
+      }
+    },
+    '& $four': {
+      left: (WIDTH + PADDING) * 3,
+      [theme.breakpoints.down('lg')]: {
+        left: '105px !important',
+        top: '65px !important',
+        transform: "rotate(15deg)"
+      }
+    },
+    '&:hover': {
+      ...revealedContent(theme),
+      [theme.breakpoints.up('lg')]: {
+        '& $revealedContent': {
+          left: 0
+        }
+      }
+    },
+  },
+  book: {
+    width: WIDTH,
+    height: HEIGHT,
+    [theme.breakpoints.down('lg')]: {
+      width: WIDTH * 0.85,
+      height: HEIGHT * 0.85,
+    },
+    [theme.breakpoints.down('sm')]: {
+      width: WIDTH * 0.7,
+      height: HEIGHT * 0.7,
+    },
+    [theme.breakpoints.down('xs')]: {
+      width: WIDTH * 0.55,
+      height: HEIGHT * 0.55,
+    },
+    borderRadius: '2px',
+    position: "absolute",
+    transition: `${transitionTime} ease`,
+    boxShadow: "-2px 2px 6px rgba(0,0,0,0.1)",
+    top: 0
+  },
+  one: {},
+  two: {},
+  three: {},
+  four: {}
+})
+
+const Book2020Animation = ({ classes, children }: {
+  classes: ClassesType,
+  children: ReactNode,
+}) => {
+  const { query } = useLocation();
+  const success = !!query.success
+  return (
+    <div className={classNames(classes.root, {[classes.success]: success})}>
+      <div className={classes.parent}>
+        <img className={classNames(classes.book, classes.one)} src="https://res.cloudinary.com/lesswrong-2-0/image/upload/v1691622592/coordination-constraint_w5obih.png" />
+        <img className={classNames(classes.book, classes.two)}src="https://res.cloudinary.com/lesswrong-2-0/image/upload/v1691622593/alignment-agency-cover_nvzux7.png" />
+        <img className={classNames(classes.book, classes.three)} src="https://res.cloudinary.com/lesswrong-2-0/image/upload/v1691622595/timelines-takeoff-cover_wnb0nc.png" />
+        <img className={classNames(classes.book, classes.four)}src="https://res.cloudinary.com/lesswrong-2-0/image/upload/v1691623651/reality-reason_llvcqx.png" />
```

Comment:
nit: pretty hard to read without any space between the `className` and `src` attributes